packages/wdio-allure-reporter/src/index.js

```diff
@@ -46,12 +46,16 @@ class AllureReporter extends WDIOReporter {
 
         const currentTest = this.allure.getCurrentTest()
 
-        const {browserName, deviceName} = this.config.capabilities
-        const targetName = browserName || deviceName || test.cid
-        const version = this.config.capabilities.version || this.config.capabilities.platformVersion || ''
-        const paramName = deviceName ? 'device' : 'browser'
-        const paramValue = version ? `${targetName}-${version}` : targetName
-        currentTest.addParameter('argument', paramName, paramValue)
+        if (!this.isMultiremote) {
+            const {browserName, deviceName} = this.config.capabilities
+            const targetName = browserName || deviceName || test.cid
+            const version = this.config.capabilities.version || this.config.capabilities.platformVersion || ''
+            const paramName = deviceName ? 'device' : 'browser'
+            const paramValue = version ? `${targetName}-${version}` : targetName
+            currentTest.addParameter('argument', paramName, paramValue)
+        } else {
```

Comment:
Can we have a test for when multiremote is true?

Reply:
@christian-bromann we have it here https://github.com/webdriverio/webdriverio/pull/3444/files#diff-7f34dcabc90a2eacc87fda48010c7a70
line `expect(addParameter).toHaveBeenCalledWith('argument', 'isMultiremote', 'true')`